-------------------------------------------------------------------------------
Test set: com.sg.classroster.dao.StudentDaoDBTest
-------------------------------------------------------------------------------
Tests run: 4, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 4.093 s <<< FAILURE! - in com.sg.classroster.dao.StudentDaoDBTest
testUpateStudent  Time elapsed: 0.514 s  <<< ERROR!
org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.sg.classroster.dao.StudentDaoDBTest.setUp(StudentDaoDBTest.java:54)
Caused by: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.sg.classroster.dao.StudentDaoDBTest.setUp(StudentDaoDBTest.java:54)

testAddAndGetStudent  Time elapsed: 1.025 s  <<< ERROR!
org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.sg.classroster.dao.StudentDaoDBTest.setUp(StudentDaoDBTest.java:54)
Caused by: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.sg.classroster.dao.StudentDaoDBTest.setUp(StudentDaoDBTest.java:54)

testGetAllStudents  Time elapsed: 1.027 s  <<< ERROR!
org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.sg.classroster.dao.StudentDaoDBTest.setUp(StudentDaoDBTest.java:54)
Caused by: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.sg.classroster.dao.StudentDaoDBTest.setUp(StudentDaoDBTest.java:54)

testDeleteStudentById  Time elapsed: 1.026 s  <<< ERROR!
org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.sg.classroster.dao.StudentDaoDBTest.setUp(StudentDaoDBTest.java:54)
Caused by: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.sg.classroster.dao.StudentDaoDBTest.setUp(StudentDaoDBTest.java:54)

